File: beecrowd/1/1009/1009.java
import java.io.IOException;
import java.util.Scanner;

class Main {

    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);
        double salario, venda;
        String nome = sc.next();
        salario = sc.nextDouble();
        venda = sc.nextDouble();
        System.out.printf("TOTAL = R$ %.2f\n", salario + (venda * 0.15));
        sc.close();

    }

}

File: beecrowd/1/1009/1009.cpp
#include <iostream>
#include <iomanip>

int main() {
    std::string nome;
    double salario, venda;
    std::cin >> nome;
    std::cin >> salario;
    std::cin >> venda;
    std::cout << std::fixed << std::showpoint;
    std::cout << std::setprecision(2) << "TOTAL = R$ " << salario + venda * 0.15 << std::endl;
    return 0;
}
